wallets/nwc/index.js

```diff
@@ -1,12 +1,26 @@
+import { Relay } from '@/lib/nostr'
+import { parseNwcUrl } from '@/lib/url'
 import { nwcSchema } from '@/lib/validate'
+import { finalizeEvent, nip04 } from 'nostr-tools'
 
 export const name = 'nwc'
 
 export const fields = [
   {
     name: 'nwcUrl',
```

Comment:
Kept this the same name to avoid having to migrate from the old name for existing wallets